src/outlet/auth-handler.ts

```diff
@@ -152,8 +155,8 @@ export class AuthHandler {
 		}
 
 		if (
-			!event?.data?.attestation
-            || !event?.data?.requestSecret
+			!event.data.hasOwnProperty('attestation')
+            || !event.data.hasOwnProperty('requestSecret')
```

Comment:
@oleggrib maybe we can just remove this check?

and replace the next lines with:
```
		this.attestationBlob = event.data?.attestation;
		this.attestationSecret = event.data?.requestSecret;
```

Reply:
we can use your code
```
!event?.data?.attestation
|| !event?.data?.requestSecret
```
and we need a check if those values are set, if not then its wrong postMessage

Reply:
Either looks good.

Reply:
I've applied your change @micwallace. Testing the full PR via Github pages examples and works well on Chrome and Safari.